src/Model/QuestionManager.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Model;
+
+class QuestionManager extends AbstractManager
+{
+    
+    const TABLE = 'question';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    /**
+     * @return array
+     * Selects a random question.
+     */
+    public function selectOneRandom(): array
+    {
+        $statement = $this->pdo->query("SELECT * FROM $this->table ORDER BY RAND() LIMIT 1");
```

Comment:
The PDO::query method can also return false (when somethings wrong) But if it does, your $statement->fetch() will cause a PHP error because you can't call a method on false.
You probably should deal with the error here

I think it's more adequate to use the self::TABLE const instead of the $table parent property to access the table name.